### PR TITLE
docs: update links for feature requests / bugs

### PR DESCRIPTION
**Pull request by braddialpad**
## Description
Created new links that go directly to our Jira for feature requests / bug reports rather than the monday.com form since we don't ever check that.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -40,11 +40,11 @@ The second is to check with the Dialtone team (in [#dialtone](https://dialpad.sl
 
 ### Feature Request
 
-If you would like to request a new feature for Dialtone, please fill out and submit [this form.](https://forms.monday.com/forms/8a9a6ff69d7e9f95caee029c2806e2c1?r=use1)
+If you would like to request a new feature for Dialtone, please submit a [feature request](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10901&pid=12428)
```
Comment (francisrupert) [inline .github/CONTRIBUTING.md]: Maybe, make it clear it's logging a Jira issue. So...

```
To request a new Dialtone feature or enhancement, submit a [feature request](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10901&pid=12428) on the Dialtone Jira project.
```

Though "Dialtone Jira project"  feels awkward.

**Review by francisrupert — APPROVED** [review]
yeh cool

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (braddialpad) [reply .github/CONTRIBUTING.md]: I suppose we don't have to say "Dialtone Jira Project". We could just say "submit a feature request to our Jira"

Comment (francisrupert) [reply .github/CONTRIBUTING.md]: 👍

**Pull request merged**